android/src/main/java/com/azendoo/reactnativesnackbar/SnackbarModule.java

```diff
@@ -73,4 +92,19 @@ public void onClick(View v) {
 
         snackbar.show();
     }
+
+    private void recursiveLoopChildren(ViewGroup view) {
```

Comment:
Interesting, seems like it should work. I'm also curious why the snackbar doesn't appear on top of the modal by default, does React Native do something sneaky to make modals appear? A link or something would be nice if you happen to know. I'm just curious.